spec/requests/families_spec.rb

```diff
@@ -1,6 +1,59 @@
 require "rails_helper"
 
 RSpec.describe "Families", type: :request do
-  describe "GET /families" do
+  let(:partner) { create(:partner) }
+  let!(:user) { create(:user, partner: partner) }
+
+  before do
```

Comment:
I tend to avoid `before` blocks like this for many of the same reasons that I avoid `let` blocks.